src/azure-cli/azure/cli/command_modules/acs/_loadbalancer.py

```diff
@@ -0,0 +1,101 @@
+    if sku:
+        return sku
+    if kubernetes_version and StrictVersion(kubernetes_version) < StrictVersion("1.13.0"):
+        print('Setting load_balancer_sku to basic as it is not specified and kubernetes \
+        version(%s) less than 1.13.0 only supports basic load balancer SKU\n' % (kubernetes_version))
+        return "basic"
+    return "standard"
+
+
+def update_load_balancer_profile(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
+                                 outbound_ports, idle_timeout, profile):
+    """parse and update an existing load balancer profile"""
+    if not is_load_balancer__profile_provided(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
+                                              outbound_ports, idle_timeout):
+        return profile
+    return configure_load_balancer_profile(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
+                                           outbound_ports, idle_timeout, profile)
+
+
+def create_load_balancer_profile(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
+                                 outbound_ports, idle_timeout):
+    """parse and build load balancer profile"""
+    if not is_load_balancer__profile_provided(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
+                                              outbound_ports, idle_timeout):
+        return None
+
+    profile = ManagedClusterLoadBalancerProfile()
+    return configure_load_balancer_profile(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
+                                           outbound_ports, idle_timeout, profile)
+
+
+def configure_load_balancer_profile(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes, outbound_ports,
+                                    idle_timeout, profile):
+    """configure a load balancer with customer supplied values"""
+    if not profile:
+        return profile
+
+    outbound_ip_resources = _get_load_balancer_outbound_ips(outbound_ips)
+    outbound_ip_prefix_resources = _get_load_balancer_outbound_ip_prefixes(outbound_ip_prefixes)
+
+    if managed_outbound_ip_count:
+        profile.managed_outbound_ips = ManagedClusterLoadBalancerProfileManagedOutboundIPs(
+            count=managed_outbound_ip_count
+        )
+    if outbound_ip_resources:
+        profile.outbound_ips = ManagedClusterLoadBalancerProfileOutboundIPs(
+            public_ips=outbound_ip_resources
+        )
+    if outbound_ip_prefix_resources:
+        profile.outbound_ip_prefixes = ManagedClusterLoadBalancerProfileOutboundIPPrefixes(
+            public_ip_prefixes=outbound_ip_prefix_resources
+        )
+    if outbound_ports:
+        profile.allocated_outbound_ports = outbound_ports
+    if idle_timeout:
+        profile.idle_timeout_in_minutes = idle_timeout
+    return profile
+
+
+def is_load_balancer__profile_provided(managed_outbound_ip_count, outbound_ips, ip_prefixes,
```

Comment:
double "_" between balancer and profile. in purpose?

Reply:
Accidental, thanks for catching. Fixing